dependency/dependency_test.go

```diff
@@ -239,6 +229,34 @@ func (c *ClientSet) createConsulTestResources() error {
 	return nil
 }
 
+func (c *ClientSet) createConsulSamenessGroups(name, partition, failoverPartition string) error {
+	members := append([]api.SamenessGroupMember{}, api.SamenessGroupMember{
+		Partition: partition,
+	}, api.SamenessGroupMember{
+		Partition: failoverPartition,
+	})
+	sg := &api.SamenessGroupConfigEntry{
+		Kind:               QuerySamenessGroup,
+		Name:               name,
+		Partition:          partition,
+		DefaultForFailover: true,
+		Members:            members,
+	}
+	log.Printf("sameness group: %+v", sg)
+	_, _, err := c.consul.client.ConfigEntries().Set(sg, &api.WriteOptions{})
+	if err != nil {
+		return err
+	}
+
+	sgs, _, err := c.consul.client.ConfigEntries().List(QuerySamenessGroup, &api.QueryOptions{})
+	if err != nil {
+		return err
+	}
+	log.Printf("sameness groups: %+v", sgs)
+
+	return nil
+}
```

Comment:
For a realistic scenario, should we create the sameness group in both partitions with the members reversed?